llvm/test/tools/UpdateTestChecks/update_mc_test_checks/Inputs/amdgpu_dasm_sort.txt.expected

```diff
@@ -0,0 +1,8 @@
+# NOTE: Assertions have been autogenerated by utils/update_mc_test_checks.py UTC_ARGS: --sort
+# RUN: llvm-mc -triple=amdgcn -mcpu=gfx1100 -disassemble -show-encoding %s 2>&1 | FileCheck -check-prefixes=CHECK %s
+
+0x00,0x00,0x00,0x7e
+# CHECK: v_nop                                   ; encoding: [0x00,0x00,0x00,0x7e]
```

Comment:
I can't tell from this test case. Does the disassembler sort on the asm output, or the hex input? I would think the asm output would be preferable

Reply:
For now all sort base on the input text so for dasm it sorts on the hex input

If we want to sort base on the asm output, what should we do with the error case?

Reply:
Hey Joe I checked the dasm and it seems sorting with hex mostly cannot group same instruction to same place, since the first byte are gprs

If we want to sort on output, it probably a bit over complicated seems we may have multiple out asm mixed with errors.

What about passing a sorting mask? It seems most of the dasm file has same length of bytes. Assume we pass a sorting mask with "0xFF0000", then for each line, i.e. "0xa,0xb,0xc", the script can merge them to 0x0a0b0c then use the mask to get the key for sorting. Do you think this could be a ugly solution? also @kosarev

Reply:
Perhaps you could exclude disasm file sorting from this patch? It seems like the other cases are working well, and we don't need to do it all at once. I don't see how sorting on the hex is going to be useful. The bits that encode the opcode would make sense to sort on, but that varies so much from instruction to instruction it will be horrible to maintain. With sorting on the output, can you redirect errors to null? Or maybe we separate those test files with both passing and error tests?

Reply:
done. dasm sorting is removed from this patch